easybuild/easyconfigs/a/AOMP/AOMP-13.0-2-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,45 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Authors:: Jorgen Nordmoen <jorgen.nordmoen@usit.uio.no>
+# License:: GPLv2 or later, MIT, three-clause BSD.
+# $Id$
+##
+
+name = 'AOMP'
+version = '13.0-2'
+
+homepage = 'https://github.com/ROCm-Developer-Tools/aomp'
+description = "AMD fork of LLVM, setup for OpenMP offloading to Accelerators"
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = ['https://github.com/ROCm-Developer-Tools/aomp/releases/download/rel_%(version)s/']
+sources = ['aomp-%(version)s.tar.gz']
+patches = ['build_aomp-%(version)s.patch']
+checksums = [
+    '0256a84aefcbf7c49112f068321b84e0620f4c43a9d490c64d820e40658e0d67',  # aomp-13.0-2.tar.gz
+    '1deca48ec55e74b091731e48ce54097b88620062ed0cb05de1dc9d56c2016a21',  # build_aomp-13.0-2.patch
+]
+
+builddependencies = [
+    ('CMake', '3.18.4'),
+    ('Perl', '5.32.0'),
+    ('Python', '3.8.6'),
+    ('elfutils', '0.183'),
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('binutils', '2.35'),
+    ('libffi', '3.3'),
+    ('ncurses', '6.2'),
+    ('numactl', '2.0.13'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/clang', 'bin/ld.lld', 'include/omp.h'],
+    'dirs': ['amdgcn', 'bin', 'include', 'lib'],
```

Comment:
Ideally, this is done in the AOMP easyblock, and a custom sanity check command is added too?

It also helps (humans) to keep the entries alphabetically sorted.

Reply:
Updated with easybuilders/easybuild-easyblocks#2464